packages/ra-ui-materialui/src/KeyboardShortcut.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import {
+    ComponentsOverrides,
+    styled,
+    SxProps,
+    Typography,
+} from '@mui/material';
+import clsx from 'clsx';
+
+export const KeyboardShortcut = ({
+    className,
+    keyboardShortcut,
+    ...rest
+}: KeyboardShortcutProps) => {
+    if (!keyboardShortcut) {
+        return null;
+    }
+
+    return (
+        <Root
+            className={clsx(KeyboardShortcutClasses.root, className)}
+            {...rest}
+        >
+            {keyboardShortcut
+                .split('>')
+                .map((sequence, sequenceIndex, sequences) => (
+                    <React.Fragment key={sequence}>
```

Comment:
nitpick: you will end up with a `Encountered two children with the same key` error if the sequence uses the same key twice, e.g. `g>g`. Not sure how you can avoid that, though. So feel free to ignore for now.